projects/pastanaga-angular/src/lib/styles/_layout.scss

```diff
@@ -21,6 +37,13 @@
     }
 }
 
+@media (min-width: $size-viewport-medium-min) {
```

Comment:
don't duplicate the media queries, add the content of this one in the same one above

Reply:
I didn't, github is showing it twice;

`@media (min-width: $size-viewport-medium-min) {
    .pa-main-container-tight {
        margin: 0 $margin-formula-tight-medium;
        max-width: calc(100vw - #{$margin-formula-tight-medium}*2);
    }
    .pa-main-container-wide {
        margin: 0 $margin-formula-wide-medium;
        max-width: calc(100vw - #{$margin-formula-wide-medium}*2);
    }
    .pa-child-container-compact {
        margin: 0 auto;
        max-width: $max-width-tight-compact;
    }
}`